packages/block-library/src/site-title/block.json

```diff
@@ -31,7 +31,11 @@
 		"html": false,
 		"color": {
 			"gradients": true,
-			"link": true
+			"link": true,
+			"__experimentalDefaultControls": {
```

Comment:
This block has a "Make title link to home" setting. Should we add `link` as one of the defaults?